target_athena/formats.py

```diff
@@ -0,0 +1,46 @@
+"""Methods for writinig different object formats."""
```

Comment:
Nice. I love this refactoring, especially how small and easy to read this file is now.